.travis.yml

```diff
@@ -1,16 +1,29 @@
+# https://travis-ci.org/sqlalchemy-redshift/sqlalchemy-redshift
+sudo: false
 language: python
-python: 3.5
+python: "3.6"
+
 env:
   matrix:
-  - TOX_ENV=py26
-  - TOX_ENV=py27
-  - TOX_ENV=py33
-  - TOX_ENV=py34
-  - TOX_ENV=py35
-  - TOX_ENV=lint
+    - TOXENV=py27
+    - TOXENV=py34
+    - TOXENV=py35
```

Comment:
just remove this line

Reply:
Remove what?

Reply:
This line

Reply:
I don't understand. What do you want me to remove and why? Do you want to stop testing on py35?

Reply:
Please delete this line. You don't need to add it here then exclude it again on line 21

Reply:
Please be more specific. What line? If I delete `TOXENV=py35` then we're no longer testing on Python 3.5. I'm fine with that personally but I don't think it's what you want.

Reply:
That's not true. Try deleting this line and the exclude block and see what happens